runtime/kusama/src/xcm_config.rs

```diff
@@ -125,6 +127,9 @@ match_types! {
 	pub type OnlyParachains: impl Contains<MultiLocation> = {
 		MultiLocation { parents: 0, interior: X1(Parachain(_)) }
 	};
+	pub type SystemParachains: impl Contains<MultiLocation> = {
+		MultiLocation { parents: 0, interior: X1(Parachain(1000)) }
```

Comment:
it would be nice maybe to reuse `pub const Statemine: MultiLocation`

Reply:
I don't know if this is possible, since `match_types!` expands to a `matches!` macro, and so this entire syntax here is actually a pattern and not an expression, hence `Statemine::get()` shouldn't be able to be evaluated as `1000`.

Reply:
Yeah, it doesn't work: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=70b79e7efda191255101e05c24e2dce3

Reply:
Should bridge hub be included?

Reply:
aha, ok, good point, 

my intention was just to reuse const instead of redefine multiple times the same thing, similar like here is also mentioned: https://github.com/paritytech/polkadot/pull/7005/files#r1157406655

more for the future, if we add more system parachains, we will need to take care of more similar constants, but it is not something crucial

Reply:
> Should bridge hub be included?

yes, should, but I have TODOs for that later, maybe after we onboard BridgeHubPolkadot